packages/repository-tests/src/crud/relations/fixtures/models/order.model.ts

```diff
@@ -24,14 +29,15 @@ export class Order extends Entity {
   @property({
     type: 'boolean',
     required: false,
+    default: false,
   })
   isShipped: boolean;
 
   @belongsTo(() => Customer)
-  customerId: number;
+  customerId: string;
```

Comment:
As I mentioned earlier, this should use `number | string` type, because many databases use numeric ids. Please fix this issue in all other places defining PK/FK properties.